content-src/components/DiscoveryStreamComponents/DSLinkMenu/DSLinkMenu.jsx

```diff
@@ -63,7 +63,12 @@ export class _DSLinkMenu extends React.PureComponent {
           onUpdate={this.onMenuUpdate}
           onShow={this.onMenuShow}
           options={TOP_STORIES_CONTEXT_MENU_OPTIONS}
-          site={{url: this.props.url, title: this.props.title, type: this.props.type}} />
+          site={{
+            referrer: "https://getpocket.com/recommendations",
```

Comment:
Eventually this value should be coming from feeds, can be a separate bug to consolidate it for DS

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1533170